docs/general/emulator-support-and-issues.md

```diff
@@ -221,29 +247,30 @@ description: Comprehensive guide to emulator support and compatibility for Retro
 ### GameCube
 
 - ✅ Standalone emulator: **[Dolphin](https://retroachievements.org/download.php#dolphin)**
-- ⌛ libretro core: **Dolphin**
-  - Testing
+  - You MUST use version 2407-68 or newer.
+  - Please make sure that the "Enable Dual Core (speedup)" option is unchecked as that mode is not supported.
```

Comment:
Recommend "Ensure "Enable Dual Core (speedup)" option is unchecked as it is unsupported."